Cargo.toml

```diff
@@ -46,7 +50,7 @@ sha2 = "0.10"
 shadow-rs = "0.19.0"
 strum = { version = "0.25", features = ["derive"] }
 thiserror = "1.0"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1", features = ["full"], default-features = false }
```

Comment:
why do we want to turn on the tokio default-features?

Reply:
I don't have any clear insight into why the project chose to have full tokio features enabled. The PR moves the `default-features = false` into the workspace toml. If we have it inside of the verifier `Cargo.toml`, it is completely ignored, and it throws a warning.